hazelcast/src/main/java/com/hazelcast/instance/DefaultNodeExtension.java

```diff
@@ -124,6 +125,12 @@ private void checkSecurityAllowed() {
                 throw new IllegalStateException("Security requires Hazelcast Enterprise Edition");
             }
         }
+        SymmetricEncryptionConfig symmetricEncryptionConfig = node.getConfig().getNetworkConfig().getSymmetricEncryptionConfig();
```

Comment:
This is a bit tricky, we just don't allow the use of the feature, but the actual mechanism is still there for anyone to use. I think we could do better, by having this mechanism live in EE.

Reply:
If you mean by mechanism the encryption inbound/outbound handlers, then they live in the EE under package `com.hazelcast.nio.tcp`.

Reply:
Yes, my mistake. I misread the issue as if someone could use symmetric on OS. Looks good to me!